library/spdm_responder_lib/libspdm_rsp_encap_get_digests.c

```diff
@@ -78,9 +78,10 @@ libspdm_return_t libspdm_process_encap_response_digest(
         status = libspdm_handle_encap_error_response_main(
             spdm_context,
             spdm_response->header.param1);
-        if (LIBSPDM_STATUS_IS_ERROR(status)) {
-            return status;
-        }
+        /* if the Requester delivers an encapsulated ERROR message, the Responder shall
+         * continue the current encapsulated communication and not move to the next request. */
+        *need_continue = true;
```

Comment:
need_continue should be assigned only if SUCCESS.

Reply:
I have updated, please check.